.github/libs/GitUtils.js

```diff
@@ -38,14 +38,11 @@ function getMergeLogsAsJSON(fromRef, toRef) {
             // Remove any double-quotes from commit subjects
             let sanitizedOutput = stdout.replace(/(?<="subject": ").*(?="})/g, subject => subject.replace(/"/g, "'"));
 
-            // Also remove any newlines
-            sanitizedOutput = sanitizedOutput.replace(/(\r\n|\n|\r)/gm, '');
+            // Also remove any newlines and escape backslashes
+            sanitizedOutput = sanitizedOutput.replace(/(\r\n|\n|\r)/gm, '').replace('\\', '\\\\');
```

Comment:
When i tested it i actually had `.replace('\\"}', '\\\\"}').replace('\\', '\\\\')`. But then i didn't understand the point of the first replace with curly brackets so i just made it generic to escape all backslashes. Let me know if I'm missing something here franc

Reply:
Yeah I think makes sense. I was trying to fix specifically for the case where a commit message ends with a backslash, but they can make the JSON string invalid regardless of where in the commit message they are.